pkg/config/config.go

```diff
@@ -521,6 +525,18 @@ func (c *LibpodConfig) Validate() error {
 	if !filepath.IsAbs(c.VolumePath) {
 		return fmt.Errorf("volume path must be an absolute path - instead got %q", c.VolumePath)
 	}
+
+	// Check if the pullType from containers.conf is valid
+	// if it is invalid returns the error
+	validPullType := map[string]bool{
```

Comment:
Can we move the map out of the function? This way we allocate memory for it only once and not for each invocation.